README.rst

```diff
@@ -17,5 +17,5 @@ Open edX Platform Integration
 
 .. code-block:: bash
 
-   $ python manage.py lms --settings test test organizations.tests
+   $ paver test_system -s lms -t organizations
```

Comment:
i think app name should also be updated here to new name: ``paver test_system -s lms -t edx_solutions_organizations``